build-farm/make-adopt-build-farm.sh

```diff
@@ -105,7 +114,6 @@ echo "OS: ${OPERATING_SYSTEM}"
 echo "SCM_REF: ${SCM_REF}"
 OPTIONS=""
 
-EXTENSION=""
```

Comment:
As long as the removal of this doesn't cause a problem when the variable is used within `sign-releases.sh` I'm ok with this - and I don't think it will.

Reply:
I checked and it doesn't seem to as EXTENSION is always set by `sign-releases.sh`:
https://github.com/AdoptOpenJDK/openjdk-build/blob/cf47cbc28665a8707485c49c58095c740a80fa43/build-farm/sign-releases.sh#L23-L30